cglib/src/main/java/net/sf/cglib/core/AbstractClassGenerator.java

```diff
@@ -75,6 +75,9 @@ public Object apply(AbstractClassGenerator gen) {
         };
 
         public ClassLoaderData(ClassLoader classLoader) {
+            if (classLoader == null) {
```

Comment:
True, I just digged a little and it seems like all classes are loaded by the system class loader in such cases.